isb_web/main.py

```diff
@@ -111,24 +113,36 @@ async def get_thing(
         raise fastapi.HTTPException(
             status_code=404, detail=f"Thing not found: {identifier}"
         )
-    if full:
+    if full or format == "full":
```

Comment:
Wasn't sure if we wanted to delete the full parameter now that its functionality has been replaced by `format`, but I left it in place in case there were backwards compatibility issues.

Reply:
Let's leave it for now since the prototype ui uses it. Will drop this parameter once ui is switched to new format